Gems/Atom/TestData/TestData/Materials/StandardPbrTestCases/011_Emissive.material

```diff
@@ -1,6 +1,6 @@
 {
     "materialType": "Materials\\Types\\StandardPBR.materialtype",
-    "materialTypeVersion": 3,
+    "materialTypeVersion": 5,
```

Comment:
Hm, why are some of these StandardPbr materials getting irradiance.irradianceColorSource set to Manual (like sponza_mat_arch.material) but many of these in StandardPbrTestCases are not getting that value set? Maybe you need to do another pass of updating the .material files after you've finished all your code changes?

Reply:
I tried to preserve the behaviour of (only) those materials that already had an explicit irradiance.color defined, hence the occasional Manual irradianceColorSource for those materials.

For materials that *didn't* already have an explicit irradiance.color, I figure that the new basecolor-derived default is better than trying to preserve the old default behaviour (which would be equivalent to setting a white manual color), i.e. it's mild breakage, but "in a good way(TM)".